x-pack/plugins/enterprise_search/public/applications/app_search/components/api_logs/components/api_logs_table.tsx

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useValues, useActions } from 'kea';
+
+import {
+  EuiBasicTable,
+  EuiBasicTableColumn,
+  EuiBadge,
+  EuiHealth,
+  EuiButtonEmpty,
+  EuiEmptyPrompt,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedRelative } from '@kbn/i18n/react';
+
+import { convertMetaToPagination, handlePageChange } from '../../../../shared/table_pagination';
+
+import { ApiLogsLogic } from '../index';
+import { ApiLog } from '../types';
+import { getStatusColor } from '../utils';
+
+import './api_logs_table.scss';
+
+interface Props {
+  hasPagination?: boolean;
+}
+export const ApiLogsTable: React.FC<Props> = ({ hasPagination }) => {
+  const { dataLoading, apiLogs, meta } = useValues(ApiLogsLogic);
+  const { onPaginate } = useActions(ApiLogsLogic);
+
+  const columns: Array<EuiBasicTableColumn<ApiLog>> = [
+    {
+      field: 'http_method',
+      name: i18n.translate('xpack.enterpriseSearch.appSearch.engine.apiLogs.table.column.method', {
+        defaultMessage: 'Method',
+      }),
+      width: '100px',
+      render: (method: string) => <EuiBadge color="primary">{method}</EuiBadge>,
+    },
+    {
+      field: 'timestamp',
+      name: i18n.translate('xpack.enterpriseSearch.appSearch.engine.apiLogs.table.column.time', {
+        defaultMessage: 'Time',
+      }),
+      width: '20%',
+      render: (dateString: string) => <FormattedRelative value={new Date(dateString)} />,
```

Comment:
So.... FYI on this, when I `mount` with this component I get a bunch of console warn's in the terminal about `FormattedRelative`:

<img width="1145" alt="" src="https://user-images.githubusercontent.com/549407/113223929-1d790f00-923f-11eb-9390-77a7925eced6.png">
<img width="1149" alt="" src="https://user-images.githubusercontent.com/549407/113223930-1eaa3c00-923f-11eb-8a89-592e933016ee.png">

Not totally sure what to do here as it's not our component or warning to fix - we can either just ignore them (they don't cause tests to fail) or silence `console.warn` for this test suite temporarily. Any preference?

Reply:
FWIW, I did a quick search in GH and saw someone mention this issue as far back as 2019: https://github.com/elastic/kibana/issues/38642#issuecomment-566553102. Looks like it either still hasn't been cleaned up in `react-intl`, or we haven't updated to the latest version of that dep. I don't know which it is.

I also saw this issue where someone works around this by mocking FormattedRelative: https://github.com/elastic/kibana/pull/91740#discussion_r578621372.

```
jest.mock('@kbn/i18n/react', () => {
  const originalModule = jest.requireActual('@kbn/i18n/react');
  const FormattedRelative = jest.fn().mockImplementation(() => '20 hours ago');

  return {
    ...originalModule,
    FormattedRelative,
  };
});
```

You could do something similar to this. That would let you drop all of the date mocking as well.

Reply:
Niiiice! I'll do that 👍

Reply:
https://github.com/elastic/kibana/pull/96008/commits/cf897c4aaef858b71d6d8e13022d527c17ace85f